library/sinks/HTTPRequest.ts

```diff
@@ -164,6 +175,14 @@ export class HTTPRequest implements Wrapper {
 
     for (const module of modules) {
       hooks.addBuiltinModule(module).onRequire((exports, pkgInfo) => {
+        if (isVersionGreaterOrEqual("16.17.0", getSemverNodeVersion())) {
```

Comment:
Does the import of subscribe breaks in Node v16.0.0? Maybe increase minimum engine version in package.json?